infra/base-images/base-runner/run_fuzzer

```diff
@@ -149,7 +149,9 @@ elif [[ "$FUZZING_ENGINE" = honggfuzz ]]; then
   # -f: location of the initial (and destination) file corpus
   # -n: number of fuzzing threads (and processes)
   CMD_LINE="$OUT/honggfuzz -n 1 --exit_upon_crash -R /tmp/${FUZZER}_honggfuzz.report -W $FUZZER_OUT -v -z -P -f \"$CORPUS_DIR\" $(get_dictionary) $* -- \"$OUT/$FUZZER\""
-
+  if [[ "$FUZZER" == *"netdriver"* ]]; then
```

Comment:
@robertswiecki 
What's the best way to identify honggfuzz target binaries that aren't using persistent mode (not sure if the term is different in honggfuzz, sorry) ?

Reply:
Looking for `HonggfuzzNetDriver_main` in the binary worked in my case, run_fuzzer distinguishes between normal honggfuzz drivers and netdriver drivers.